code/arrays/arrays.py

```
@@ -699,7 +699,18 @@ def pop(self) -> any:
+    
+    def remove(self, val: Any) -> None:
+        """Remove first occurrence of value (or raise ValueError)."""
+        for k in self._n:
+            if self._A[k] == val:
+                for j in range(k, self._n, -1):
+                    self.A[j] = self.A[j+1]
+                self.A[self._n] = None
+                self._n -= 1
+                return
+            
+        raise ValueError('Element not found')
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)